config/manifests/kustomization.yaml

```diff
@@ -1,4 +1,27 @@
+# These resources constitute the fully configured set of manifests
+# used to generate the 'manifests/' directory in a bundle.
 resources:
+- bases/clowder.clusterserviceversion.yaml
 - ../default
 - ../samples
 - ../scorecard
+
+# [WEBHOOK] To enable webhooks, uncomment all the sections with [WEBHOOK] prefix.
+# Do NOT uncomment sections with prefix [CERTMANAGER], as OLM does not support cert-manager.
+# These patches remove the unnecessary "cert" volume and its manager container volumeMount.
+#patchesJson6902:
+#- target:
```

Comment:
should we uncomment this?

Reply:
I'm really not sure

Reply:
works in minikube, will see if it works in olm